.github/workflows/pr.yaml

```diff
@@ -54,7 +54,7 @@ jobs:
       build_type: pull-request
       package-name: ucx_py
       test-docker-options: "--cap-add CAP_SYS_PTRACE --shm-size=8g"
-      test-before-arm64: "pip install cupy-cuda11x -f https://pip.cupy.dev/aarch64"
+      test-before-arm64: "pip install cupy-cuda11x<12.0.0 -f https://pip.cupy.dev/aarch64"
```

Comment:
I think this doesn't make sense. `cupy-cuda11x` has CUDA 11 in the name because it's supposed to be CUDA 11 only. Am I missing something?